nservicebus/sqlserver/deployment-options_multi-instance_sqlserver_[,3].partial.md

```diff
@@ -1,8 +1,11 @@
 ### Multi-instance
 
-WARNING: The *multi-instance* option won't be supported in Versions 4 and higher. Version 3 supports *multi-instance* mode only with DTC.
+WARNING: The *multi-instance* option won't be supported in Versions 4 and higher.
```

Comment:
>WARNING: The *multi-instance* option won't be supported in Versions 4 and higher.

should this be:

WARNING: The *multi-instance* option is no longer supported from Versions 4 and higher.